app/controllers/application_controller.rb

```diff
@@ -32,10 +32,13 @@ def user_for_paper_trail
     user_signed_in? ? current_user.id : 'Guest'
   end
 
+  def default_url_options
+    # need heroku config:set HOST=www.ebwiki.org
+    { host: ENV['HOST'] || 'localhost:3000' }
```

Comment:
Let's put the second string in a value in `config/locales/en.yml`, perhaps in a value named `en.development.host` ?

Reply:
do you mean adding this to en.yml:

```
  development:
    host: 'localhost:3000'
```
and adjusting the method to pul from that file?

Reply:
i Added this to the yaml file, but can't actually  pull the value into the controller,